package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "contracts",
   "devDependencies": {
-    "@0xpolygonid/js-sdk": "1.14.1",
-    "@iden3/js-jsonld-merklization": "1.2.0",
-    "@iden3/contracts": "^2.3.0",
+    "@0xpolygonid/js-sdk": "1.20.0",
+    "@iden3/js-jsonld-merklization": "1.4.1",
+    "@iden3/contracts": "../../iden3/contracts/contracts",
```

Comment:
You can put 2.4.4 version here already

Reply:
Yes, I thought I did it in my previous commit but missing to push `package.json`. Done it.